subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/concurrent/IO.kt

```diff
@@ -69,8 +69,19 @@ interface AsyncIOScopeFactory {
 }
 
 
-internal
-inline fun <T> withAsynchronousIO(
+fun <T> withAsynchronousIO(
```

Comment:
Is this now part of the public API or is it still internal?

Reply:
Still "internal", this still holds https://docs.gradle.org/current/userguide/kotlin_dsl.html#sec:implicit_imports